lib/rbac/filterer.rb

```diff
@@ -367,6 +367,10 @@ def apply_rbac_directly?(klass)
       CLASSES_THAT_PARTICIPATE_IN_RBAC.include?(safe_base_class(klass).name)
     end
 
+    def apply_rbac_by_scope_in?(klass)
+      %w(CustomButtonEvent).include?(klass.name)
```

Comment:
Shouldn't this be defined in the class instead of hardcoding class names here?

Reply:
Or, it could be something like `klass.respond_to?(:scope_for_model)`. But, if you go that route, the method should have a more specific name like `rbac_scope_for_model`